frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -14,8 +14,8 @@ import {
   MapProps as LeafletMapProps,
   TileLayer,
   Popup,
-  LayersControl,
-  WMSTileLayer,
+  // LayersControl,
```

Comment:
Not sure if these will be used later or not, but just in case you forgot to remove